### PR TITLE
.NET Interactive: Enable_try_dotnet_to_batch_10b

### PR DESCRIPTION
**Pull request by v-mepa**
Enable_try_dotnet_to_batch_10b

https://docs.microsoft.com/en-us/dotnet/api/system.exception | Batch 10 | Can be enabled   for first and third examples
-- | -- | --
https://docs.microsoft.com/en-us/dotnet/api/system.type | Batch 10 | Remaining examples   will work with the new syntax
https://docs.microsoft.com/en-us/dotnet/api/system.object.gethashcode | Batch 10 | Except sample 4
https://docs.microsoft.com/en-us/dotnet/api/system.random.next | Batch 10 | Remaining examples   will work with the new syntax
https://docs.microsoft.com/en-us/dotnet/api/system.enum | Batch 10 | Only for the full   samples that have the using statements
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.-ctor | Batch 10 | Remaining examples   will work with the new syntax
https://docs.microsoft.com/en-us/dotnet/api/system.string.-ctor | Batch 10 | Only examples tagged   as Example 1, 2 and 3 can be added. Others use unsafe keyword.
https://docs.microsoft.com/en-us/dotnet/api/system.string.normalize | Batch 10 | Examples seem   duplicate. Maybe move them to the MemberGroup area?
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.lastindexof | Batch 10 | Duplicate samples for   all overloads? Potentially move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.queue-1.-ctor | Batch 10 | Duplicate samples for   some overloads? Potentially move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.string.isnormalized | Batch 10 | Examples seem   duplicate. Maybe move them to the MemberGroup area?

### CODE REVIEW

**Comment by v-mepa** [comment]
fyi @mairaw

File: xml/System/String.xml [hunk 1]
```diff
@@ -7954,6 +7954,17 @@ The `comparisonType` parameter is a <xref:System.StringComparison> enumeration m
...
+
+## Examples
+ The following example determines whether a string is successfully normalized to various normalization forms.
+
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/string.normalize/CPP/norm.cpp" id="Snippet1":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/string.normalize/CS/norm.cs" interactive="try-dotnet" id="Snippet1":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR/string.normalize/VB/norm.vb" id="Snippet1":::
+
+ ]]></format>
       </Docs>
```
Comment (mairaw) [inline xml/System/String.xml:7968]:
```suggestion
        </remarks>
      </Docs>
```

File: xml/System/String.xml [hunk 2]
```diff
@@ -9953,6 +9947,18 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
       </AssemblyInfo>
       <Docs>
         <summary>Returns a new string whose binary representation is in a particular Unicode normalization form.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Examples
+ The following example normalizes a string to each of four normalization forms, confirms the string was normalized to the specified normalization form, then lists the code points in the normalized string.
+
+ [!code-cpp[string.normalize#1](~/samples/snippets/cpp/VS_Snippets_CLR/string.normalize/CPP/norm.cpp#1)]
```
Comment (mairaw) [inline xml/System/String.xml:9956]: You need to change to the new syntax here

File: xml/System/String.xml [hunk 3]
```diff
@@ -9936,9 +9930,9 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
 ## Examples
  The following example demonstrates the <xref:System.String.Length%2A> property.
 
- [!code-cpp[string.length#1](~/samples/snippets/cpp/VS_Snippets_CLR/string.length/CPP/length.cpp#1)]
- [!code-csharp-interactive[string.length#1](~/samples/snippets/csharp/VS_Snippets_CLR/string.length/CS/length.cs#1)]
- [!code-vb[string.length#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/string.length/VB/length.vb#1)]
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/string.length/CPP/length.cpp" id="Snippet1":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/string.length/CS/length.cs" interactive="try-dotnet" id="Snippet1":::
```
Comment (mairaw) [inline xml/System/String.xml:9934]:
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/string.length/CS/length.cs" interactive="try-dotnet-method" id="Snippet1":::
```

File: xml/System/String.xml [hunk 4]
```diff
@@ -9936,9 +9930,9 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
 ## Examples
  The following example demonstrates the <xref:System.String.Length%2A> property.
 
- [!code-cpp[string.length#1](~/samples/snippets/cpp/VS_Snippets_CLR/string.length/CPP/length.cpp#1)]
- [!code-csharp-interactive[string.length#1](~/samples/snippets/csharp/VS_Snippets_CLR/string.length/CS/length.cs#1)]
- [!code-vb[string.length#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/string.length/VB/length.vb#1)]
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/string.length/CPP/length.cpp" id="Snippet1":::
```
Comment (mairaw) [inline xml/System/String.xml:9933]: weird... this change made the C# and C++ samples disappear now 🤔 
Maybe we should roll back the change for these 3 lines

**Comment by mairaw** [comment]
Before you do that, let me try to rebuild this.... the other API was also impacted.

**Comment by v-mepa** [comment]
Sure, let me know.

**Pull request merged**